src/main/java/net/dancier/chatdancer/dao/ChatDao.java

```diff
@@ -0,0 +1,20 @@
+package net.dancier.chatdancer.dao;
+
+import net.dancier.chatdancer.models.Chat;
+import net.dancier.chatdancer.models.Message;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface ChatDao {
+
+    Chat createNewChatByParticipants(Chat chat);
```

Comment:
by the name of the method I expect participants as parameters. Maybe just `createNewChat`?